Blish HUD/GameServices/Input/Keyboard/TypedInputUtil.cs

```diff
@@ -110,6 +110,9 @@ internal static class TypedInputUtil {
                 // Single character in buffer
                 case 1:
                     ret = sbString[0].ToString();
+                    if (shift) {
```

Comment:
Can we mark this with a `//TODO: ` or something to indicate that we need to review in the future?  I'll create an issue to track it since certainly we'll want to aim for better keyboard layout support in the future, but I think this could be a fine solution for now.  Better than no support.

Reply:
I've already committed a patch handling all the issues I've found so far.